force-app/main/default/lwc/link/link.html

```diff
@@ -1,5 +1,5 @@
 <template>
-    <slot onclick={handleClick}>
+    <slot onclick={handleClick} onslotchange={handleSlotChange}>
```

Comment:
I have install this on my org, but for me `onslotchange` event not calling after changing the Link.

![image](https://user-images.githubusercontent.com/27341550/92746205-abbf9d80-f3a0-11ea-9593-d51a60d139b7.png)

In the above image, On the load its properly setting the active attribute to 'item one' but after clicking on 'item two', Active attribute should change to Link 'item two'.

Please review at your end.

Thanks :)

Reply:
Good catch!   Forgot to set the active flag in the route subscription callback ( this.setCurrentPath() ).
That ensures that the active state is updated when any link causes the navigation event.

Reply:
Awesome :) It's working now very well.